### ISSUE TITLE
load have problem with includes on relative path

### ISSUE THREAD

**Issue by jeleniste** [geographika/mappyfile#60]
It looks (if I'm not doing something wrong way), that loads don`t work well with relative paths in include. On debian.

` mpf=mappyfile.open("cesky-rudolec/cesky-rudolec.map")
`
Works well
but
```f=codecs.open("cesky-rudolec/cesky-rudolec.map", "r", "utf-8") 
mpf=mappyfile.load(f)                                         
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/mappyfile/utils.py", line 59, in load
    ast = p.load(fp)
  File "/usr/local/lib/python2.7/dist-packages/mappyfile/parser.py", line 163, in load
    return self.parse(text)
  File "/usr/local/lib/python2.7/dist-packages/mappyfile/parser.py", line 185, in parse
    text = self.load_includes(text, fn=fn)
  File "/usr/local/lib/python2.7/dist-packages/mappyfile/parser.py", line 70, in load_includes
    raise ex
IOError: [Errno 2] No such file or directory: '/gis/mapserv/../include/include_head.map'
```

**Comment by geographika** [comment]
@jeleniste - there is a difference in how ``open`` and ``load`` work. ``open`` has a filename, and uses this 
to set the working directory from which to find includes (see parser.py):

```
# Per default use working directory of the process
    if fn is None:
        fn = os.getcwd() + os.sep
```

When using ``load`` a file handle is passed with no filename so the ``cwd`` defaults to the current ``cwd``.

Maybe try setting the ``cwd`` before opening the file?
```
os.chdir(os.path.dirname(fn))
f = open(fn)
mpf=mappyfile.load(f) 
```

**Comment by geographika** [comment]
Although it does seem possible to get the filename from a filehandle (with ``f.name``), so I'll consider updating the code to automatically do this.

**Comment by jeleniste** [comment]
@geographika  Thanks

**Comment by geographika** [comment]
@jeleniste - this should be fixed in the latest 0.7.1 release available on pypi
